### DISCUSSION TITLE
Migrating from MySQL library to Plex/PKC?

### DISCUSSION THREAD

**Discussion by thesmj** [croneter/PlexKodiConnect#1834] [Q&A]
Is it possible to import the data from a Kodi/MySQL database over to Plex?

What I'm thinking might work:

1. On Kodi, export the library to .db
2. Install PKC on Kodi
3. Break the MySQL connection.
4. Once Kodi/PKC is working, import the .db file.
5. Success?

**Comment by croneter** [comment]
You do not import your existing MySQL data into PlexKodiConnect or Kodi, but into Plex. PlexKodiConnect then takes the Plex data automatically and plugs it into Kodi..

Hence: google differently 😉 

Edit: I don't really find anything on "migrating from Kodi to Plex". Try firing up Plex, point it to your media folders - voilà. For the watched status, try something like using Trakt with your existing Kodi. Then try importing Trakt watched status into Plex.

**Reply by thesmj** [reply]
> For the watched status, try something like using Trakt with your existing Kodi. Then try importing Trakt watched status into Plex.

That's what I'm trying to do - migrate the watched status that's currently in MySQL on Kodi over to Plex, so I can drop MySQL.

I'll give this a try thanks!

EDIT: Just checked into it. There's a $30 fee to sync Plex with Trakt. I might consider it if it's the only option, but given that I'd only be using Trakt once to migrate watched status I'd rather keep looking for other options.
